adc/microphone_adc/microphone_adc.c

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/gpio.h"
+#include "hardware/adc.h"
+#include "hardware/uart.h"
+#include "pico/binary_info.h"
+
+ /* Example code to plot analog values from a microphone
```

Comment:
This `C` file doesn't do any plotting, so perhaps this comment needs tweaking?

I guess it depends whether the comment here is referring to the whole project, or just this individual file within the project? :shrug: ping @aallan

Reply:
I've tweaked it now to explain what does the plotting and what does the ADC value reading.